hallux/tools/factory.py

```diff
@@ -12,7 +12,7 @@
 from ..tools.sonarqube.solver import Sonar_IssueSolver
 
 
-class ProcessorFactory:
+class ToolsFactory:
     @staticmethod
     def init_solvers(
```

Comment:
Refactor this function to reduce its Cognitive Complexity from 28 to the 15 allowed.
```suggestion
    def init_solvers(
```